boards/eval_l073z.json

```diff
@@ -0,0 +1,32 @@
+{
+  "build": {
+    "cpu": "cortex-m0plus",
+    "extra_flags": "-DSTM32L073xx",
+    "f_cpu": "32000000L",
+    "mcu": "stm32l073vzt6"
+  },
+  "frameworks": [
+    "mbed",
```

Comment:
I don't see this board listed in mbed support. How do you use it? With stm32cube?

Reply:
The most parameters were copied from L053 Disco or Nucleo board. I am using stm32cube, no experience with mbed.